nsdb-cluster/src/multi-jvm/scala/io/radicalbit/nsdb/cluster/actor/MetadataTest.scala

```diff
@@ -78,6 +78,8 @@ class MetadataTest extends MultiNodeSpec(MetadataTest) with STMultiNodeSpec with
 
   val metadataCoordinator = system.actorSelection("/user/guardian/metadata-coordinator")
 
+  metadataCoordinator ! WarmUpLocations(List.empty)
```

Comment:
why do we need to notify the warm up explicitly in the multi-jvm tests in which there is supposed to be recreated the whole actors lifecycle?